src/plone/restapi/interfaces.py

```diff
@@ -84,6 +84,38 @@ def __call__(value):
         """
 
 
+class IBlockFieldDeserializationTransformer(Interface):
+    """Convert/adjust raw block deserialized value into block value.
+    """
+
+    order = Attribute("A number used in sorting value transformers. "
```

Comment:
You should add block_type as an attribute to the interface class

Reply:
Good catch, thank you!